src/main/java/org/folio/circulation/domain/representations/AccountStorageRepresentation.java

```diff
@@ -31,6 +33,8 @@ public AccountStorageRepresentation(Loan loan, Item item, FeeFineOwner feeFineOw
     this.put("loanId", loan.getId());
     this.put("userId", loan.getUserId());
     this.put("itemId", item.getItemId());
+    write(this, "dueDate", loan.getDueDate());
```

Comment:
Why not use `JsonPropertyWriter` to write the rest of the fields?

Reply:
We could, but it wasn't required for this particular task so I left the rest of the fields as they are. For new fields `JsonPropertyWriter` was a better choice because they represent date-time and it's better to keep the date formatting logic in one place. I could switch the rest of the fields to `JsonPropertyWriter` too if you think it'll improve something.